src/index.js

```diff
@@ -141,6 +142,8 @@ export function form(fn, config = {}) {
     config
   );
 
+  storeValue.update(value => ({ ...value, initCheck: config.initCheck }));
```

Comment:
Am I missing something or could this not just be added to \`storeValue\` right away :thinking: (so move `config` above `const storeValue = ...` (L128)

Reply:
Can be done @AnthyG , I did not want to move the structure much, that's why I haven't moved it.